packages/radix-ui-themes/src/components/scroll-area.css

```diff
@@ -1,9 +1,15 @@
-.rt-ScrollAreaRoot {
-  --scrollarea-scrollbar-margin-top: var(--space-1);
-  --scrollarea-scrollbar-margin-bottom: var(--space-1);
-  --scrollarea-scrollbar-margin-left: var(--space-1);
-  --scrollarea-scrollbar-margin-right: var(--space-1);
+.radix-themes {
+  --scrollarea-scrollbar-horizontal-margin-top: var(--space-1);
+  --scrollarea-scrollbar-horizontal-margin-bottom: var(--space-1);
+  --scrollarea-scrollbar-horizontal-margin-left: var(--space-1);
+  --scrollarea-scrollbar-horizontal-margin-right: var(--space-1);
+  --scrollarea-scrollbar-vertical-margin-top: var(--space-1);
+  --scrollarea-scrollbar-vertical-margin-bottom: var(--space-1);
+  --scrollarea-scrollbar-vertical-margin-left: var(--space-1);
+  --scrollarea-scrollbar-vertical-margin-right: var(--space-1);
+}
```

Comment:
Moved the scrollarea vars here so that we don't need to target the scrollbar element to override them. This way, with consistent selector specificity, whether the overrides work doesn't hinge on the source order.